ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -64,6 +64,10 @@
             new AccelerateDecelerateInterpolator();
     private static final int DEFAULT_GRAVITY = Gravity.START;
 
+    public static final int DIRECTION_ANY = 0;
```

Comment:
I'm tempted to make this an enum instead of consts, thoughts?

Reply:
I actually started doing it the enum way, but reverted when I saw the other `DEFAULT_xxx` values on the `TickerView`. It would have the added compile-time safety benefit when calling `setPreferredScrollingDirection`... If you want me to go ahead and do that I can

Reply:
let's use enum here, i've never understood why the framework used int consts for stuff like gravity and visibility, and ive never been a huge fan of it myself haha.

Reply:
Sounds good to me, I'll have a fix pushed momentarily